Exception/ResponseException.php

```diff
@@ -23,6 +23,37 @@
  */
 class ResponseException extends ConsumerException
 {
+
+    /**
+     * Flag for message ack
+     */
+    const ACK = 1;
+
+    /**
+     * Flag single for message nack and requeue
+     */
+    const NACK_REQUEUE = 2;
+
+    /**
+     * Flag for reject and requeue message to the same spot in main queue
+     */
+    const REJECT_REQUEUE = 3;
+
+    /**
+     * Flag for reject and drop message
+     */
+    const REJECT = 4;
+
+    /**
+     * Flag for reject and resend message to the end of main queue
+     */
+    const REJECT_RESEND = 5;
```

Comment:
To logically connect it to REJECT_REQUEUE I suggest to rename this constant to REJECT_REQUEUE_AS_NEW